### PR TITLE
adding EU enviroment to docs

### PR DESCRIPTION
**Pull request by Verlich**
JIRA: https://jira.inmoment.com/browse/DEV-19777

### CODE REVIEW

File: source/includes/_api_index.md [hunk 1]
```diff
@@ -26,6 +26,17 @@ You can use our API to access and update your end users, responses and declines.
 
 All responses from the API (including errors) are in JSON. All code examples are in cURL.
 
+# Environments
+
+Currently Wootric supports two main Environments <strong> EU </strong> and <strong> US </strong>.
+
+All examples provides in this section are for <strong> US </strong> environment only, if you want
```
Comment (diegoserranoa) [inline source/includes/_api_index.md]: `All examples provides` or `All examples provided`? :)

**Review by diegoserranoa — COMMENTED** [review]
Looks good @Verlich ! Thanks for taking care of this. I will only ask @DanPitrowiski to helps us checking the wording is the best one here. Can you help @DanPitrowiski ? 🙏 

Screen Shot 2022-03-07 at 9 29 09 PM

**Comment by DanPitrowiski** [comment]
@pmonfort @diegoserranoa just made a few edits with the help of Grammarly (great tool for spell checking). Looks great! Sorry I didn't just add the comments into the code. I thought it would help to write it out like this.

> Currently, Wootric supports two main environments: the **EU** and **US**.
> 
> All examples provided in this section are for the US environment only. If you want to run these examples on the EU environment, you need to replace the **.com** domain with **.eu**.
> 
> To determine if your account runs on the EU environment now, check if your account token begins with **NPS-EU**.

**Comment by Verlich** [comment]
@DanPitrowiski @diegoserranoa Thanks for your review! Just updated it :D

**Comment by DanPitrowiski** [comment]
@Verlich sorry I tagged Pablo. 🙃  Glad the changes worked.

**Review by diegoserranoa — APPROVED** [review]

**Pull request merged**